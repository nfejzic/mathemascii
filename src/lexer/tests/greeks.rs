use super::Snapshot;

super::test_snap!(greek_pi, "pi");
super::test_snap!(greek_alpha, "alpha");
super::test_snap!(multiple, "alphabetagammadelta");
