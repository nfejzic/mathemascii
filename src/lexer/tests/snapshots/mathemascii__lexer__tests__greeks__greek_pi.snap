---
source: src/lexer/tests/greeks.rs
expression: "Snapshot((input, tokens))"
---
pi

pi
^^ -> Greek(Pi) at: 0 -> 2


